napari/layers/labels/_tests/test_labels.py

```diff
@@ -279,14 +279,12 @@ def test_colormap():
     np.random.seed(0)
     data = np.random.randint(20, size=(10, 15))
     layer = Labels(data)
-    assert type(layer.colormap) == tuple
-    assert layer.colormap[0] == 'random'
-    assert type(layer.colormap[1]) == Colormap
+    assert type(layer.colormap) == Colormap
```

Comment:
I'm not sure if we have plans to subclass `Colormap`, but should we use `isinstance` instead?

```suggestion
    assert isinstance(layer.colormap, Colormap)
```

Reply:
fixed here and everywhere